tools/zuul-projects-checks.py

```diff
@@ -170,6 +172,42 @@ def check_only_boilerplate():
     return errors
 
 
+def check_project_templates():
+    untrusted_github_projects = ansible_tenant["source"]["github.com"][
+        "untrusted-projects"
+    ]
+    github_projects = [
+        "github.com/" + i for i in untrusted_github_projects if type(i) == str
+    ]
+
+    errors = False
+    for github_project in github_projects:
+        templates = []
+        for project in [i["project"] for i in projects]:
+            name = project["name"]
+            if name.startswith("^") and re.match(name, github_project):
+                templates += project.get("templates", [])
+            elif github_project == name:
+                templates += project.get("templates", [])
+        if not templates:
+            print(
+                "No template for project {github_project}".format(
+                    github_project=github_project
+                )
+            )
+            errors = True
+        if "noop-jobs" not in templates:
```

Comment:
we no longer need to check for noop-jobs, just system-required.

Reply:
Ok.